### PR TITLE
Extract DV2 dispatcher resource requirements to a configuration variable.

### PR DESCRIPTION
**Pull request by tmooney**

### CODE REVIEW

**Comment by ebelter** [comment]
+1

File: lib/perl/Genome/Model/Tools/DetectVariants2/Dispatcher.pm [hunk 1]
```diff
@@ -102,6 +102,9 @@ class Genome::Model::Tools::DetectVariants2::Dispatcher {
         lsf_queue => {
             default_value => Genome::Config::get('lsf_queue_dv2_workflow'),
         },
+        lsf_resource => {
+            default => Genome::Config::get('lsf_resource_dv2_dispatcher'),
```
Comment (acoffman) [inline lib/perl/Genome/Model/Tools/DetectVariants2/Dispatcher.pm]: What is the difference between the `default` here and the `default_value` immediately above it? Anything?

Comment (ebelter) [inline lib/perl/Genome/Model/Tools/DetectVariants2/Dispatcher.pm]: These are synonyms: default, value and default_value. The default and value both normalize to default_value.

Comment (acoffman) [inline lib/perl/Genome/Model/Tools/DetectVariants2/Dispatcher.pm]: Gotcha. I might advocate for keeping the label consistent within the same class header, but I don't feel strongly either way.

**Comment by acoffman** [comment]
+1

File: lib/perl/Genome/Model/Tools/DetectVariants2/Dispatcher.pm (hunk 1 above)
Comment (tmooney) [inline lib/perl/Genome/Model/Tools/DetectVariants2/Dispatcher.pm]: I've changed the line that said `default_value` to match the other five that are easily seen in this PR. :smile:

**Pull request merged**